src/lib/route-file-content.js

```diff
@@ -60,8 +60,9 @@ function routeFileContent(location, callback) {
       );
     }
 
-    var Index = require('../pages/index.jsx');
-    FS.makeTree(Path.join(__dirname, '..', 'public', location)).then(function() {
+    var Index = require('../pages/index.js');
+    var dirPath = Path.join(__dirname, '..', '..', 'public', location);
```

Comment:
I needed to change the path because things are now one more dir in, so I needed to `..` an extra step. I initially only did this path change only in the `nameOfTheFile` below, and missed it on the `FS.makeTree` line. I had to do extra debugging to catch this, so for the future, I now define the path in one spot as `dirPath` so I only need to manage the path in one spot. :)